src/components/popovers/menu.test.js

```diff
@@ -12,20 +18,20 @@ describe('Popover component', () => {
    */
   it('matches snapshot (string items)', () => {
     const wrapper = mount(
-      <Menu trigger={<button>trigger</button>}>
-        <MenuItem>First</MenuItem>
-        <MenuItem>Second</MenuItem>
-        <MenuItem>Third</MenuItem>
+      <Menu trigger={<a href="#">trigger</a>}>
+        <MenuItem onSelect={() => {}}>First</MenuItem>
+        <MenuItem onSelect={() => {}}>Second</MenuItem>
+        <MenuItem onSelect={() => {}}>Third</MenuItem>
       </Menu>
     );
     expect(wrapper).toMatchSnapshot();
   });
   it('matches snapshot (JSX items)', () => {
     const wrapper = mount(
-      <Menu trigger={<button>trigger</button>}>
-        <MenuItem><p>First</p></MenuItem>
-        <MenuItem><p>Second</p></MenuItem>
-        <MenuItem><p>Third</p></MenuItem>
+      <Menu trigger={<a href="#">trigger</a>}>
+        <MenuItem onSelect={() => {}}><p>First</p></MenuItem>
+        <MenuItem onSelect={() => {}}><p>Second</p></MenuItem>
+        <MenuItem onSelect={() => {}}><p>Third</p></MenuItem>
```

Comment:
Why no button here anymore?

Reply:
also, we could utilize storybook actions here instead of noop potentially?

Reply:
native `button` inside of a button is no bueno - avoids a warning

Reply:
agree about using actions - Filed a follow up, UIP-626